src/methods/send.ts

```diff
@@ -1,25 +1,20 @@
-import { GenomeBrowserType } from '../types';
-import { OutgoingAction, OutgoingActionType } from '../types';
+import {
+  OutgoingActionType,
+  type GenomeBrowserType,
+  type OutgoingAction,
+  type BrowserSetFocusAction
+} from '../types';
+import {  } from '../types';
```

Comment:
What is being imported here?

Reply:
I must have been in the process of moving all imports from the second import statement into the first one; and forgot to delete the import that I moved everything out of 😄